app/plugin_requests_test.go

```diff
@@ -4,23 +4,67 @@
 package app
 
 import (
+	"fmt"
+	"io"
 	"net/http"
 	"net/http/httptest"
+	"os"
+	"path/filepath"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/mattermost/mattermost-server/v6/utils/fileutils"
 )
 
 func TestServePluginPublicRequest(t *testing.T) {
+	t.Skip()
```

Comment:
@lieut-data Just to double check: Are you planing to enable the test in this PR?

Reply:
Thanks for checking, @hanzei! I'm trying to figure out why running this test leaves the system in a state that breaks other tests. Definitely don't want to leave it disabled. This has also been broken for some time, so I'd be fine with deprioritizing for v7.7.

Reply:
@hanzei, fixed! Was the plugin clean up issue as I expected, I just needed to tweak the approach.